hawc/apps/lit/templates/lit/overview.html

```diff
@@ -59,6 +59,7 @@ <h3>Literature summary</h3>
             <th>Untagged</th>
             <td>{{overview.total_untagged}}</td>
           </tr>
+
```

Comment:
can you revert the changes in this file? just delete the extra lines which were added.

Reply:
Sorry but I don't understand what you mean here? I'm not sure which extra lines were added, unless you mean just delete white space?

Reply:
the green lines in the diffs were changes that you made to these files; it looks like you added some whitespace in this file and another file. you can delete those lines so that when we look at the "file changes" tab these files no longer appear